src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -99,6 +102,8 @@ const text = {
   [SPEED_UP]: `This will speed up your pending transaction by replacing it. There’s still a chance your original transaction will confirm first!`,
 };
 
+let existingWallet;
```

Comment:
Is there a reason this needs to be declared here?

Reply:
Can't be declared in the sheet component scope cause the sheet gets unmounted after speeding up and the state is gone.